index.html

```diff
@@ -2595,6 +2595,119 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>External Resource Integrity</h2>
+        <p>
+          When including a link to an external resource in a VC, it is 
+          desirable to know whether the resource that is pointed to is the
+          same at signing time as at verification time.  This applies
+          to cases where there is an external resource that is
+          remotely retrieved as well as to cases where the issuer and/or
+          verifier may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
+          context(s) used in the verifiable credential are the same when
+          used by both the issuer and verifier. 
+        </p>
+        <p>
+          To validate that a resource referenced by a Verifiable Credential is
+          the same at verification time as at issuing time, an implementer
+          MAY include a property named <code>resourceIntegrity</code> that
+          stores an array of objects that describe additional integrity
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
+          is present, there MUST be an object in the array for each remote
+          resource.
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the URL to the resource named <code>id</code>, a
+          <code>timestamp</code>
+          that indicates the time at which the hash was computed, the
+          <code>digest</code>
+          of the resource, and the <code>method</code> which indicates what
+          hashing algorithm was used as listed as the 'Hash Name String'
+          property from the <a
+          href="https://www.iana.org/assignments/named-information/named-information.xhtml">IANA
+          Named Information Hash Algorithm Registry</a>.  
+          The <code>digest</code> property MUST be the base64url [[RFC 4648]]
+          encoded digest of the hash.
```

Comment:
If you're going to cite RFC 4648 rather than RFC 7515, you also need to say that the encoding has no trailing = characters, since 4648 doesn't say to omit them.